src/TobiasOlry/TalklyBundle/Command/UserCreateCommand.php

```diff
@@ -29,8 +34,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $io = new SymfonyStyle($input, $output);
 
-        $username = $io->ask('username');
-        $password = $io->askHidden('password');
+        if (!$username = $input->getArgument('username')) {
```

Comment:
I thought this should normally be done in the `interact()` method (see http://symfony.com/doc/current/components/console/introduction.html#command-lifecycle)